src/js/SelectFields/SelectField.js

```diff
@@ -581,7 +588,9 @@ export default class SelectField extends Component {
     const menuProps = {
       isOpen: open,
       close: this.close,
-      className: classnames('md-select-field-menu-container', menuClassName),
+      className: classnames('md-select-field-menu-container', menuClassName, {
+        'full-width': {fullWidth}
```

Comment:
this line should just be `'full-width': fullWidth` The way it is now is always truthy